index.html

```diff
@@ -16,14 +16,17 @@
   <body>
     <header>
       <nav>
-        <a href="#">My Logo</a>
+        <a href="#" id="logo">My Logo</a>
         <div class="link">
-          <ul>
-            <li><a href="#">Portfolio</a></li>
-            <li><a href="#">About</a></li>
-            <li><a href="#">Contact</a></li>
-          </ul>
-          <img src="./assets/menu.png" alt="logo" />
+          <div class="nav-link" id="navLink">
+            <img src="assets/close.png" alt="close" onclick="hideMenu()" />
+            <ul>
+              <li id="Portfolio" onclick="hideMenu()"><a href="#portfolio">Portfolio</a></li>
+              <li id="About-me" onclick="hideMenu()"><a href="#about">About</a></li>
+              <li id="Contact-me" onclick="hideMenu()"><a href="#contact">Contact</a></li>
+            </ul>
+          </div>
+          <img src="./assets/menu.png" id="menu" alt="logo" onclick="showMenu()" />
```

Comment:
* Currently when I click on the hamburger icon on mobile it doesn't show the mobile menu dropdown, kindly fix this